test/user.test.js

```diff
@@ -401,6 +401,29 @@ describe('User', function() {
     var pass73Char = pass72Char + '3';
     var passTooLong = pass72Char + 'WXYZ1234';
 
+    it('rejects empty passwords creation', function(done) {
+      try {
+        User.create({email: 'b@c.com', password: ''}, function(err) {
+          if (err) return done(err);
+          done(new Error('User.create() should have thrown an error.'));
+        });
+      } catch (e) {
```

Comment:
This is pretty weird. A callback/promise-based function should report errors via callback/promise rejection, not by throwing synchronously to the caller.

```js
User.create({email: 'b@c.com', password: ''}, function(err) {
  expect(err).to.have.property('code', 'INVALID_PASSWORD');
  done();
});
```